pkg/apis/openebs.io/v1alpha1/cstor_volume.go

```diff
@@ -35,24 +36,25 @@ type CStorVolume struct {
 
 // CStorVolumeSpec is the spec for a CStorVolume resource
 type CStorVolumeSpec struct {
-	Capacity          string `json:"capacity"`
-	TargetIP          string `json:"targetIP"`
-	TargetPort        string `json:"targetPort"`
-	Iqn               string `json:"iqn"`
-	TargetPortal      string `json:"targetPortal"`
-	Status            string `json:"status"`
-	NodeBase          string `json:"nodeBase"`
-	ReplicationFactor int    `json:"replicationFactor"`
-	ConsistencyFactor int    `json:"consistencyFactor"`
+	Capacity          resource.Quantity `json:"capacity"`
```

Comment:
Since this is API i.e. schema, we should provide enough comments that explains the field with defaults and sample values and so on.

Reply:
added comment to capacity